packages/jest-environment/src/index.ts

```diff
@@ -148,6 +149,11 @@ export interface Jest {
    * Not available when using legacy fake timers implementation.
    */
   getRealSystemTime(): number;
+  /**
+   * Fetches the seed generated for this particular running of Jest.
+   * It can also be manually set with --seed CLI argument.
```

Comment:
```suggestion
   * Retrieves the seed value. It will be randomly generated for each test run
   * or can be manually set via the `--seed` CLI argument.
```